core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -108,16 +108,7 @@ public boolean isDisabled() {
                 Strings.isNullOrEmpty(getContext().getInstrumentationKey()) || configuration.isTrackingDisabled();
     }
 
-    /**
-     * Sends the specified state of a user session to Application Insights.
-     * @param sessionState {@link com.microsoft.applicationinsights.telemetry.SessionState}
-     *                     value indicating the state of a user session.
-     */
-    public void trackSessionState(SessionState sessionState) {
```

Comment:
True. Pulling the whole API could break users. Maybe mark it @Depreciated and change the implementation to call trackEvent() instead?